### DISCUSSION TITLE
Slampher R2 cannot unpair remote.  Red light never flashes

### DISCUSSION THREAD

**Discussion by bwims** [arendst/Tasmota#16757] [Q&A] [answered]
Hello,

I have a Slampher R2 which is running Tasmota v11.0.0 (though it has been upgraded several times)

Shortly after flashing,  I found it convenient to pair a remote with it.

However, I would like to redeploy the remote and find that I cannot unpair it.   Whatever I do, the red light will not flash.

Is there a software solution?

Many thanks

B.

**Comment by barbudor** [comment]
Unfortunately Tasmota has no relationship with the RF part that is managed by a separated MCU.
If you can't make that MCU forget about your remote, you may have no other solution than cutting a track or removing a part to sever that MCU

**Comment by barbudor** [comment, marked as answer]
From my understanding the MCU triggers gpio0 which is tasmota 's button
If you are not using the on-device button, change the template of gpio from button to line and tasmota will ignore orders from the rf

**Reply by bwims** [reply]
Hi,  many thanks for your reply!

I could not find a "line" option for the template, so I set GPIO0 to "None".  Didn't work, however,  Could I change it to anything else?

**Reply by barbudor** [reply]
Sorry I meant none. Don't know how 'line' came here.
So you mean that despite having no more button configured on the template, the RF still controls the lamp?
And what about the on device button?
If the on device button is still working, then you probably didn't saved/applied the change correctly.

**Reply by bwims** [reply]
How odd!  The button does still work!

Is it sufficient to simply add the template, or do I need to edit the module?  This is what I have saved:

Module

![image](https://user-images.githubusercontent.com/18516073/194707009-454be772-5b93-4720-b8fa-7c2a49cf36a2.png)

Template

![image](https://user-images.githubusercontent.com/18516073/194707026-fb06f84b-e341-4da1-9b16-8c12a0ef373a.png)

**Comment by bwims** [comment]
I worked around it.  I just used the Generic (18) and copied the Relay and Led settings, leaving GPIO0 at "none" and it worked!

Thanks SO much!  I'm **very** grateful!

**Reply by barbudor** [reply]
Was about to suggest the same 👍
